sandy/decay.py

```diff
@@ -148,6 +156,39 @@ def get_decayconstant(self, with_uncertainty=True):
         else:
             return df.LAMBDA
 
+    def custom_perturbation_decayconstant(self, pert, zam="all"):
+        """
+        Apply a custom perturbation to the decay constant
+        Parameters
+        ----------
+        zam : `int` or `list` or `all`, default is `all`
+            ZAM number of the material to which perturbation is to be
+            applied.
+        pert : 'float'
+            Perturation coeffcient as ratio value.
+        Returns
+        -------
+        `sandy.decay.DecayData`
+            DacayData instance with given value ZAM and decay constant perturbed.
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33",'decay',[922350, 922330, 942390, 942320, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> tape =sandy.DecayData.custom_perturbation_decayconstant(rdd, 0.05, [922350, 922330])
+        >>> print(tape)
+        922350   3.27689e-17
+        922330   1.44821e-13
+        Name: LAMBDA, dtype: float64
+        """
+        tape = sandy.DecayData.get_decayconstant(self)
+        if zam != 'all':

```

Comment:
I don't like the 'all' option. We are never in the situation of perturbing all decay constants.
I would either have it not ad default, or even better remove it....it will simplify the method

Reply:
I wrote the 'all' option because I supposed that the user can have different isotopes and can decide to perturb a specific one (I didn't mean all the decay constants from the decay data) but surely  he can use a filter to do this. If you think that this option is not valid I will remove it.